examples/dmd/parametric_tw_csv.cpp

```diff
@@ -16,15 +16,15 @@
 // =================================================================================
 //
 // Parametric serial DMD command (for HDF version, append -hdf):
-//   mpirun -np 8 parametric_tw_csv -o hc_parametric_serial -rdim 16 -dtc 0.01 -offline
-//   mpirun -np 8 parametric_tw_csv -o hc_parametric_serial -rdim 16 -dtc 0.01 -online
+//   mpirun -np 8 parametric_tw_csv -o hc_parametric_serial -rdim 16 -dtc 0.01 -offline -snap-pfx step
+//   mpirun -np 8 parametric_tw_csv -o hc_parametric_serial -rdim 16 -dtc 0.01 -online -snap-pfx step
 //
 // Final-time prediction error (Last line in run/hc_parametric_serial/dmd_par5_prediction_error.csv):
 //   0.0012598331433506
 //
 // Parametric time windowing DMD command (for HDF version, append -hdf):
-//   mpirun -np 8 parametric_tw_csv -o hc_parametric_tw -nwinsamp 25 -dtc 0.01 -offline
-//   mpirun -np 8 parametric_tw_csv -o hc_parametric_tw -nwinsamp 25 -dtc 0.01 -online
+//   mpirun -np 8 parametric_tw_csv -o hc_parametric_tw -nwinsamp 25 -dtc 0.01 -offline -snap-pfx step
+//   mpirun -np 8 parametric_tw_csv -o hc_parametric_tw -nwinsamp 25 -dtc 0.01 -online -snap-pfx step
```

Comment:
Should we make the default value `snap_pfx = "step"` so that this input argument can be skipped in the sample run?

Reply:
The default option `snap_pfx = ""` works when there is no prefix, i.e. the full filename is written in the list. If we make the default `snap_pfx = "step"`, is there a way to make the option as empty `char` array when the executable is run? I try to write nothing after `-snap_pfx`, but it returns `Missing argument for the last option: -snap-pfx`.

Reply:
Does `-snap_pfx ""` work?

Reply:
It works. I changed the default value.